docs/cli/argo_archive_list.md

```diff
@@ -6,6 +6,22 @@ list workflows in the archive
 argo archive list [flags]
 ```
 
+### Examples
+
+```
+# List all Archived Workflows:
+  argo archive list
+
```

Comment:
codegen is [erroring](https://github.com/argoproj/argo-workflows/actions/runs/6724019736/job/18275387013?pr=12110) on the whitespace here because you have trailing whitespace in the code, but not in the generated docs.

Can you get `make codegen` to run on its own (i.e. without running `make pre-commit`)?
that will make things a lot simpler than trying to manually match up to what `codegen` expects